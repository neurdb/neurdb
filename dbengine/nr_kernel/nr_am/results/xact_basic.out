psql: error: connection to server on socket "/tmp/.s.PGSQL.5432" failed: FATAL:  the database system is in recovery mode
