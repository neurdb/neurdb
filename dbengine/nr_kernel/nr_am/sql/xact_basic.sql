BEGIN;
CREATE TABLE x(a INT, b text, c text, PRIMARY KEY (a, b)) USING nram;
-- Test insert and table scan.
SELECT * from x;
INSERT INTO x VALUES (1, 'k1', 'v1');
SELECT * from x;
COMMIT;

BEGIN;
SELECT * from x;
INSERT INTO x VALUES (2, 'k2', 'v2');
SELECT * from x;
DROP TABLE x;
COMMIT;
