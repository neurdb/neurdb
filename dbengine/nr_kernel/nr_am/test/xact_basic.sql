BEGIN;
CREATE TABLE x(a INT, b text, c text, PRIMARY KEY (a, b)) USING nram;
-- Test insert and table scan.
SELECT * from x;
INSERT INTO x VALUES (1, 'k1', 'v1');
SELECT * from x;
DROP TABLE x;
COMMIT;
