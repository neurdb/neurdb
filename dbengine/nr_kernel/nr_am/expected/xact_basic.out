BEGIN;
CREATE TABLE x(a INT, b text, c text, PRIMARY KEY (a, b)) USING nram;
-- Test insert and table scan.
SELECT * from x;
 a | b | c 
---+---+---
(0 rows)

INSERT INTO x VALUES (1, 'k1', 'v1');
SELECT * from x;
 a | b  | c  
---+----+----
 1 | k1 | v1
(1 row)

DROP TABLE x;
COMMIT;
